app/assets/javascripts/events.coffee

```diff
@@ -1,34 +1,34 @@
-class @EventsController
-  index: ->
-    $('#event').select2
-      minimumInputLength: 1
-      allowClear: true
-      placeholder: "ブキ検索"
+# class @EventsController
+#   index: ->
+#     $('#event').select2
+#       minimumInputLength: 1
+#       allowClear: true
+#       placeholder: "ブキ検索"
 
-    $('#stage').select2
-      minimumInputLength: 0
-      allowClear: true
-      placeholder: "ステージ検索"
+#     $('#stage').select2
+#       minimumInputLength: 0
+#       allowClear: true
+#       placeholder: "ステージ検索"
 
-    SearchEvents = (selectBox) ->
-      paramaters = $.param({
-        weapons: $("#event").val(),
-        stages: $("#stage").val()
-      })
+#     SearchEvents = (selectBox) ->
+#       paramaters = $.param({
+#         weapons: $("#event").val(),
+#         stages: $("#stage").val()
+#       })
 
-      url = selectBox.data("url") + "?" + paramaters
+#       url = selectBox.data("url") + "?" + paramaters
```

Comment:
__[Misspell]__

"paramaters" is a misspelling of "parameters"


<!-- Comment by [Sider](https://sider.review) -->